velox/type/Type.h

```diff
@@ -571,6 +611,14 @@ class TypeBase : public Type {
     return TypeTraits<KIND>::isFixedWidth;
   }
 
+  bool isOrderable() const override {
```

Comment:
This is not quite right.

For array type, orderable == whether element type is orderable.
For row type, orderable == whether all child types are orderable.

Same for comparable.

Reply:
@mbasmanova We should add these two methods in the logical type (instances of Type class) not in the physical type (TypeKind) in the current implementation.

Reply:
@duanmeng I think so.

CC: @pedroerp @aditi-pandit @majetideepak